test/lib/sql_annotation.py

```diff
@@ -37,7 +37,7 @@ def receive(func):
         def wrapper(*args, **kwargs):
             """wrapper"""
             name = args[1].name
-            args[0].db = list(args[1].db)
+            args[0].db = args[1].db
```

Comment:
Whether multiple DBS are supported?

Reply:
such as db[0], db[1]...

Reply:
yes, now DB init is in method init_data, this line actually does't work

Reply:
list(db) is removed, set does not support index?